src/index.ts

```diff
@@ -39,3 +39,8 @@ export {default as LoginButton} from './FBLoginButton';
 export {default as SendButton} from './FBSendButton';
 export {default as ShareButton} from './FBShareButton';
 export {default as AEMReporterIOS} from './FBAEMReporter';
+
+export type {
+  ShareContent,
+  ShareContentCommonParameters,
+} from './models/FBShareContent';
```

Comment:
Currently, we need the type `ShareContent` in the example project to avoid type error.

We might be missing more types to export excepted `ShareContent` and `ShareContentCommonParameters`, my strategy here is to only fix issues that I have encountered.